lib/alerts_viewer_web/live/bus_live.ex

```diff
@@ -150,7 +157,29 @@ defmodule AlertsViewerWeb.BusLive do
   def seconds_to_minutes(nil), do: nil
 
   def seconds_to_minutes(seconds) do
-    (seconds / 60) |> Float.round(1)
+    (seconds / 60) |> round
```

Comment:
```suggestion
    (seconds / 60) |> round()
```
nitpick: Could we please include the empty parens to make it more obvious that this is a function call?